CODEOWNERS

```diff
@@ -1,2 +1,2 @@
-*       @Yalz @rorlic @tomvda @sverholen @WLefever-Cegeka @pj-cegeka @Tomvbe
+*       @Yalz @rorlic @tomvda @sverholen @WLefever-Cegeka @pj-cegeka @Tomvbe @jobulcke
```

Comment:
Maybe point for disucssion.. since the list of code owners is getting quite long, should passive owners still be included?